enso/config.py

```diff
@@ -61,6 +61,12 @@
     # 'SequenceLabeling/brown_verbs',
     # 'SequenceLabeling/brown_pronouns',
     # 'SequenceLabeling/brown_adverbs',
+    'RationalClassify/bank_qualified',
```

Comment:
we should probably change to rationalizedclassify. sorry for being the naming stickler

Reply:
I agree